refbox/src/app/mod.rs

```diff
@@ -1817,6 +1814,53 @@ impl RefBoxApp {
                 trace!("AppState changed to {:?}", self.app_state);
                 task
             }
+            Message::AutoConfirmScores(snapshot) => {
+                info!("Autoconfirming");
+
+                let mut tasks = vec![];
+                tasks.push(self.apply_snapshot(snapshot));
+
+                self.app_state = match self.app_state {
+                    AppState::ConfirmScores(scores) => {
+                        let now = Instant::now();
+                        let mut tm = self.tm.lock().unwrap();
+                        if let Some(id) = self.schedule.as_ref().map(|schedule| &schedule.event_id)
+                        {
+                            tasks.push(self.post_game_score(id, tm.game_number(), scores));
+                        }
+
+                        tm.set_scores(scores, now);
+
+                        AppState::MainPage
+                    }
+                    AppState::ScoreEdit {
+                        scores,
+                        is_confirmation,
+                    } => {
+                        if is_confirmation {
+                            let now = Instant::now();
+                            let mut tm = self.tm.lock().unwrap();
+                            if let Some(id) =
+                                self.schedule.as_ref().map(|schedule| &schedule.event_id)
+                            {
+                                tasks.push(self.post_game_score(id, tm.game_number(), scores));
+                            }
+
+                            tm.set_scores(scores, now);
```

Comment:
The effect of this call is to apply changes before the user has pushed submit. For example, scores are tied at 1, and we enter the confirmation pause. User says no, scores aren't correct, then sets score in the dialog to 2-1, but doesn't press done. Right now, this would automatically set the scores to 2-1 at the end of the pause. I think we should just remove the call for now until we get the ref timeout changes implemented